### PR TITLE
Fixed the loaders notebook in docs to work in colab.

### PR DESCRIPTION
**Pull request by gsaluja9**
Also addresses a bug in Images.py
Need to see why all polygons do not get loaded from annotations.

### CODE REVIEW

File: examples/CocoDataPyTorch.py [hunk 1]
```diff
@@ -91,18 +95,38 @@ def generate_query(self, idx: int):
                 m = self.coco_detection.coco.annToMask(meta_info)
                 polygons = polygonFromMask(m)
                 adb_polygons = []
+                adb_boxes = []
                 for polygon in polygons:
                     adb_polygon = []
+                    left, top, right, bottom = 99999, 99999, 0, 0
```
Comment (drewaogle) [inline examples/CocoDataPyTorch.py:101]: I guess 99999 is ok for unset for coco?

Comment (gsaluja9) [reply examples/CocoDataPyTorch.py:101]: i've not seen images of that resolution in coco

**Review by drewaogle — CHANGES_REQUESTED** [review]
Can we remove the outputs from loaders.ipyndb? it makes it very hard to review.

**Review by drewaogle — APPROVED** [review]
fixing the issue with diffing noteoboks is too hard here.

**Pull request merged**